src/hotspot/share/runtime/threadSMR.cpp

```diff
@@ -366,6 +366,7 @@ class ScanHazardPtrPrintMatchingThreadsClosure : public ThreadClosure {
   }
 };
 
+#ifdef DEBUG
```

Comment:
Shouldn't that be s/DEBUG/ASSERT/ ?  And similarly for the use.

Reply:
Ugghh! Thanks Kim. Yes a mental slip after initially using DEBUG_ONLY. I guess we will never fix that inconsistency.